src/styles/variables/variables.scss

```diff
@@ -0,0 +1,16 @@
+$header-shadow: #0000000d;
+$link-color: #060b35;
+$link-color-active: #00acdc;
+$card-background-color: #fff;
+$card-border-color: #f3f3f3;
+$card-secondary-color: #616070;
+$card-main-accent-color: #060b35;
+$card-button-color: #00acdc;
+$card-tetle-color: #34568b;
+$card-price-color: #000;
+$card-width: 200px;
+$font-weight4: 400;
```

Comment:
this is the default font weight. You don't need to write it to use it

Reply:
> this is the default font weight. You don't need to write it to use it

Not fixed, and the name is not informative at all, why it's `$font-weight4`, I don't see 3, 2, 1